### ISSUE TITLE
Capitalization

### ISSUE THREAD

**Issue by Fibrolan** [RealRaven2000/SmartTemplate4#107]
I am paid subscriber of Smart Templates and Enjoy using it. I think that though that an important feature which is not included in Thunderbird would be an extremely valuable asset to your add on. This feature is capitalization of a letter right after a period when start a sentence or Paragraph.
Would there is an interest in adding this. I am sure a lot more users besides me would love having this added feature

**Comment by RealRaven2000** [comment]
It might be an idea - I think there are very few Add-ons (even if at all) that improve the experience in Composer. One thing I would also love to add (and I raise a separate issue once I see I have capacity) would be toolbar buttons to insert certain HTML things e.g. a `<span class=issueButton>New Issue</span>` - I have to do this manually quite frequently in order to direct my users to the github issues pages.

Before we can tackle this (in Thunderbird 78 only) I need to solve the problem of running content scripts in Composer via the API, because I do not want to create a most likely not compatible global keyboard listener such as the one I have in my Add-on [ZombieKeys](https://addons.thunderbird.net/en-US/thunderbird/addon/zombie-keys/).

**Comment by RealRaven2000** [comment]
I changed the title - you would most likely want capitalization after period followed by a whitespace character (space or new line). You probably wouldn't want it to capitalize something like a web URL or email address - which has letters following directly after the period character.
